src/tracks/playabletrack/wavetrack/ui/WaveTrackView.h

```diff
@@ -61,6 +61,7 @@ class AUDACITY_DLL_API WaveTrackSubView : public CommonTrackView
    std::weak_ptr<UIHandle> mResizeHandle;
    std::weak_ptr<UIHandle> mAdjustHandle;
    std::weak_ptr<UIHandle> mRearrangeHandle;
+   std::weak_ptr<UIHandle> mClipTrimHandle;
```

Comment:
std::weak_ptr<UIHandle> instead of some subclass raises the danger of the slicing problem, which you correct in a later commit.

But look at all those old bad examples before it that I wrote!  These should be corrected too.